tests/test_default.py

```diff
@@ -0,0 +1,21 @@
+def test_brew_taps(Command):
+    c = Command('/usr/local/bin/brew tap --list')
+    assert c.rc == 0
+    tapped = c.stdout.split()
+    assert 'homebrew/core' in tapped
```

Comment:
> Wouldn't this just be testing that Ansible's core modules are doing what they say they do?

Not quite-- it's for testing that the _role_ does what it says it does. Take the following scenarios:

1. A developer deletes the task that installs homebrew packages.
1. A developer changes the default cask applications such that e.g. `google-chrome` is installed rather than `firefox`.

Simply running the test playbook and the idempotence test would not catch these issues. 

Testinfra verifies that the desired state on the target host(s) has been attained. Think of it like functional tests alongside your application code. 

The idempotence test is still essential and I don't intend to downplay its significance. IMO, "run it" and "run it again" is more of smoke test.

I hope that answers your question 😄